<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>261</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Get more imformation</string>
   </property>
   <widget class="QPushButton" name="pushButton_Tiger">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiger</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_swallow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Swallow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Rabit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Rabit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mouse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Cat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Lion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Lion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Leopard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Leapard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Eagle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Eagle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Dog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_crocodil">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Crocodil</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_hello">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>671</width>
     <height>291</height>
    </rect>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="plainText">
    <string>Press the button 
To get imformations

Thank you for using this programm</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_buy">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Buy tickets</string>
   </property>
   <widget class="QPushButton" name="pushButton_buy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>buy</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
